R/cchart.R

```diff
@@ -321,9 +329,9 @@ CChart <- function(chart.type, x, small.multiples = FALSE,
                         format = "",
                         prefix = "&nbsp;",
                         suffix = "",
-                        color = user.args$data.label.color,
```

Comment:
How were the appearance font settings being applied previously? Is that what `user.args$data.label.*` was set to when labels weren't shown?

Reply:
The appearance font settings are applied [here](https://github.com/Displayr/Plugins/blob/master/src/Standard%20R/Visualization/Area/Area.R#L27). They are still used in the fallback when the javascript controls are not defined, e.g. https://github.com/Displayr/Plugins/blob/master/src/Standard%20R/Visualization/Area/Area.R#L456

Reply:
OK. Are there any situations where this would be the case?

Reply:
I realised that `user.args$global.font.size` is defined, so I'll use that, so users aren't stuck with a fixed font size (they can still adjust it via a visualization template or the appearance tab)

Reply:
Back to your original question. The appearance tab font setting are applied when the "Use default or template font" checkbox is ticked (its in the FONT group). By default its not ticked because then they can't see what fonts were selected, but I think some people do use so they can change all the fonts at once.